sites/all/themes/dcap/css/style.css

```diff
@@ -848,6 +939,31 @@ body {
 .page-faqs .view-faqs {
   padding: 5% 4.5%;
 }
+.page-colleges form.ctools-auto-submit-full-form,
+.page-faqs form.ctools-auto-submit-full-form {
+  padding: 0;
+  text-align: left;
+}
+.page-colleges form.ctools-auto-submit-full-form .form-item,
+.page-faqs form.ctools-auto-submit-full-form .form-item {
+  background: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+.page-colleges form.ctools-auto-submit-full-form .form-item input,
+.page-faqs form.ctools-auto-submit-full-form .form-item input {
+  -ms-border-radius: 8px;
```

Comment:
For instances like this where in you have browser prefixes .. use mixins so that you would not miss any browser prefix